doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -108,7 +109,12 @@ Then you need to add the ebpf flags to configure ::
  sudo mkdir /etc/suricata/ebpf/
 
 The ``clang`` compiler is needed if you want to build eBPF files as the build
-is done via a specific eBPF backend available only in llvm/clang suite.
+is done via a specific eBPF backend available only in llvm/clang suite. If you
+don't want to use Clang for building Suricata itself is not desired, you can
```

Comment:
Looks like `is not desired` is a left over of some previous version.

Reply:
Argh. :-)